<?xml version="1.0" encoding="UTF-8" ?>
<testsuite tests="34" failures="17" name="io.cloudboost.CreateTableTest" time="9.605" errors="0" skipped="0">
  <properties>
    <property name="java.runtime.name" value="Java(TM) SE Runtime Environment"/>
    <property name="sun.boot.library.path" value="C:\Program Files\jdk1.8.0_05\jre\bin"/>
    <property name="java.vm.version" value="25.5-b02"/>
    <property name="java.vm.vendor" value="Oracle Corporation"/>
    <property name="maven.multiModuleProjectDirectory" value="E:\Engineer\programming\projects\javaSDK\git_repo"/>
    <property name="java.vendor.url" value="http://java.oracle.com/"/>
    <property name="path.separator" value=";"/>
    <property name="guice.disable.misplaced.annotation.check" value="true"/>
    <property name="java.vm.name" value="Java HotSpot(TM) Client VM"/>
    <property name="file.encoding.pkg" value="sun.io"/>
    <property name="user.script" value=""/>
    <property name="user.country" value="US"/>
    <property name="sun.java.launcher" value="SUN_STANDARD"/>
    <property name="sun.os.patch.level" value=""/>
    <property name="java.vm.specification.name" value="Java Virtual Machine Specification"/>
    <property name="user.dir" value="E:\Engineer\programming\projects\javaSDK\git_repo"/>
    <property name="java.runtime.version" value="1.8.0_05-b13"/>
    <property name="java.awt.graphicsenv" value="sun.awt.Win32GraphicsEnvironment"/>
    <property name="java.endorsed.dirs" value="C:\Program Files\jdk1.8.0_05\jre\lib\endorsed"/>
    <property name="os.arch" value="x86"/>
    <property name="java.io.tmpdir" value="C:\Users\new\AppData\Local\Temp\"/>
    <property name="line.separator" value="
"/>
    <property name="java.vm.specification.vendor" value="Oracle Corporation"/>
    <property name="user.variant" value=""/>
    <property name="os.name" value="Windows 8.1"/>
    <property name="classworlds.conf" value="C:\Program Files (x86)\apache-maven-3.3.3\bin\..\bin\m2.conf"/>
    <property name="sun.jnu.encoding" value="Cp1252"/>
    <property name="java.library.path" value="C:\Program Files\jdk1.8.0_05\bin;C:\Windows\Sun\Java\bin;C:\Windows\system32;C:\Windows;C:\ProgramData\Oracle\Java\javapath;C:\oraclexe\app\oracle\product\11.2.0\server\bin;;C:\Program Files (x86)\Intel\iCLS Client\;C:\Program Files\Intel\iCLS Client\;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;C:\Program Files (x86)\GNU\GnuPG\pub;C:\Program Files (x86)\Microsoft SQL Server\100\Tools\Binn\;C:\Program Files\Microsoft SQL Server\100\Tools\Binn\;C:\Program Files\Microsoft SQL Server\100\DTS\Binn\;C:\Program Files (x86)\Skype\Phone\;C:\Program Files\nodejs\;C:\Program Files\MySQL\MySQL Server 5.6\bin;C:\Program Files (x86)\apache-ant-1.9.6\bin;C:\Program Files\jdk1.8.0_05\bin;E:\Engineer\eclipse\android-sdk-windows\tools;E:\Engineer\eclipse\android-sdk-windows\platform-tools;C:\Program Files\jdk1.8.0_05\bin;C:\Program Files\MongoDB\Server\3.0\bin;E:\Engineer\eclipse\eclipse;E:\Engineer\eclipse\myeclipse;C:\Program Files (x86)\apache-maven-3.3.3\bin;C:\Program Files (x86)\GNU\GnuPG;C:\Program Files\openssl-0.9.8k_X64\bin;C:\Program Files\nodejs;C:\Users\new\AppData\Roaming\npm;C:\Program Files\Wireshark;."/>
    <property name="java.specification.name" value="Java Platform API Specification"/>
    <property name="java.class.version" value="52.0"/>
    <property name="sun.management.compiler" value="HotSpot Client Compiler"/>
    <property name="os.version" value="6.3"/>
    <property name="user.home" value="C:\Users\new"/>
    <property name="user.timezone" value="Africa/Nairobi"/>
    <property name="java.awt.printerjob" value="sun.awt.windows.WPrinterJob"/>
    <property name="java.specification.version" value="1.8"/>
    <property name="file.encoding" value="Cp1252"/>
    <property name="user.name" value="new"/>
    <property name="java.class.path" value="C:\Program Files (x86)\apache-maven-3.3.3\bin\..\boot\plexus-classworlds-2.5.2.jar"/>
    <property name="java.vm.specification.version" value="1.8"/>
    <property name="sun.arch.data.model" value="32"/>
    <property name="java.home" value="C:\Program Files\jdk1.8.0_05\jre"/>
    <property name="sun.java.command" value="org.codehaus.plexus.classworlds.launcher.Launcher test"/>
    <property name="java.specification.vendor" value="Oracle Corporation"/>
    <property name="user.language" value="en"/>
    <property name="awt.toolkit" value="sun.awt.windows.WToolkit"/>
    <property name="java.vm.info" value="mixed mode"/>
    <property name="java.version" value="1.8.0_05"/>
    <property name="java.ext.dirs" value="C:\Program Files\jdk1.8.0_05\jre\lib\ext;C:\Windows\Sun\Java\lib\ext"/>
    <property name="sun.boot.class.path" value="C:\Program Files\jdk1.8.0_05\jre\lib\resources.jar;C:\Program Files\jdk1.8.0_05\jre\lib\rt.jar;C:\Program Files\jdk1.8.0_05\jre\lib\sunrsasign.jar;C:\Program Files\jdk1.8.0_05\jre\lib\jsse.jar;C:\Program Files\jdk1.8.0_05\jre\lib\jce.jar;C:\Program Files\jdk1.8.0_05\jre\lib\charsets.jar;C:\Program Files\jdk1.8.0_05\jre\lib\jfr.jar;C:\Program Files\jdk1.8.0_05\jre\classes"/>
    <property name="sun.stderr.encoding" value="cp437"/>
    <property name="java.vendor" value="Oracle Corporation"/>
    <property name="maven.home" value="C:\Program Files (x86)\apache-maven-3.3.3\bin\.."/>
    <property name="file.separator" value="\"/>
    <property name="java.vendor.url.bug" value="http://bugreport.sun.com/bugreport/"/>
    <property name="sun.cpu.endian" value="little"/>
    <property name="sun.io.unicode.encoding" value="UnicodeLittle"/>
    <property name="sun.stdout.encoding" value="cp437"/>
    <property name="sun.desktop" value="windows"/>
    <property name="sun.cpu.isalist" value="pentium_pro+mmx pentium_pro pentium+mmx pentium i486 i386 i86"/>
  </properties>
  <testcase classname="io.cloudboost.CreateTableTest" name="deleteCompanyTable" time="0"/>
  <testcase classname="io.cloudboost.CreateTableTest" name="updateCompanySchema" time="0"/>
  <testcase classname="io.cloudboost.CreateTableTest" name="deleteAddressTable" time="0"/>
  <testcase classname="io.cloudboost.CreateTableTest" name="createCompanyTable" time="0"/>
  <testcase classname="io.cloudboost.CreateTableTest" name="deleteEmployeeTable" time="0"/>
  <testcase classname="io.cloudboost.CreateTableTest" name="createEmployeeTable" time="0"/>
  <testcase classname="io.cloudboost.CreateTableTest" name="createAddressTable" time="0"/>
  <testcase classname="io.cloudboost.CreateTableTest" name="updateEmployeeSchema" time="0"/>
  <testcase classname="io.cloudboost.CreateTableTest" name="createTableWith2UnderscoreColumns" time="0"/>
  <testcase classname="io.cloudboost.CreateTableTest" name="deleteEmptyTable" time="0.001"/>
  <testcase classname="io.cloudboost.CreateTableTest" name="createTableRole" time="0.335"/>
  <testcase classname="io.cloudboost.CreateTableTest" name="createTableUser" time="0.33"/>
  <testcase classname="io.cloudboost.CreateTableTest" name="createEmptyTable" time="0.001"/>
  <testcase classname="io.cloudboost.CreateTableTest" name="createEmpty" time="0.361">
    <failure type="junit.framework.AssertionFailedError:">junit.framework.AssertionFailedError: null
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.Assert.assertTrue(Assert.java:20)
	at junit.framework.Assert.assertTrue(Assert.java:27)
	at io.cloudboost.CreateTableTest$4.done(CreateTableTest.java:229)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CreateTableTest.createEmpty(CreateTableTest.java:224)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CreateTableTest" name="updateCustomTable" time="0.368">
    <failure message="Cannot PUT /kisenyi/table/vygsxdpa" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /kisenyi/table/vygsxdpa
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CreateTableTest$10.done(CreateTableTest.java:495)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CreateTableTest.updateCustomTable(CreateTableTest.java:489)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CreateTableTest" name="createTableCustom14" time="0.473">
    <failure message="Cannot PUT /kisenyi/table/ckehzbxg" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /kisenyi/table/ckehzbxg
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CreateTableTest$21.done(CreateTableTest.java:901)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CreateTableTest.createTableCustom14(CreateTableTest.java:895)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CreateTableTest" name="createTableCustom1" time="0.358">
    <failure message="Cannot PUT /kisenyi/table/cdgqbbbq" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /kisenyi/table/cdgqbbbq
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CreateTableTest$22.done(CreateTableTest.java:934)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CreateTableTest.createTableCustom1(CreateTableTest.java:928)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CreateTableTest" name="createTableSample" time="0.746">
    <failure message="Cannot PUT /kisenyi/table/rkqycqiv" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /kisenyi/table/rkqycqiv
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CreateTableTest$12.done(CreateTableTest.java:577)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CreateTableTest.createTableSample(CreateTableTest.java:571)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CreateTableTest" name="createTableStudent4" time="0.001"/>
  <testcase classname="io.cloudboost.CreateTableTest" name="createTableCustom5" time="0.352">
    <failure message="Cannot PUT /kisenyi/table/lzvdfbbk" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /kisenyi/table/lzvdfbbk
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CreateTableTest$11.done(CreateTableTest.java:533)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CreateTableTest.createTableCustom5(CreateTableTest.java:527)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CreateTableTest" name="createTableStudent" time="0.37">
    <failure message="Cannot PUT /kisenyi/table/nlxjkbjz" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /kisenyi/table/nlxjkbjz
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CreateTableTest$15.done(CreateTableTest.java:682)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CreateTableTest.createTableStudent(CreateTableTest.java:676)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CreateTableTest" name="createTableCustom18" time="0.339">
    <failure message="Cannot PUT /kisenyi/table/hhdhggmh" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /kisenyi/table/hhdhggmh
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CreateTableTest$16.done(CreateTableTest.java:711)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CreateTableTest.createTableCustom18(CreateTableTest.java:705)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CreateTableTest" name="createTableCustom2" time="1.109">
    <failure message="Cannot PUT /kisenyi/table/glehvoof" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /kisenyi/table/glehvoof
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CreateTableTest$19$1$1.done(CreateTableTest.java:818)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CreateTableTest$19$1.done(CreateTableTest.java:812)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CreateTableTest$19.done(CreateTableTest.java:799)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CreateTableTest.createTableCustom2(CreateTableTest.java:795)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CreateTableTest" name="updateTableSample" time="0.001"/>
  <testcase classname="io.cloudboost.CreateTableTest" name="createTableCustom4" time="0.779">
    <failure message="Cannot PUT /kisenyi/table/dadblzzs" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /kisenyi/table/dadblzzs
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CreateTableTest$20$1.done(CreateTableTest.java:863)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CreateTableTest$20.done(CreateTableTest.java:857)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CreateTableTest.createTableCustom4(CreateTableTest.java:847)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CreateTableTest" name="createTablestudent1" time="1.15">
    <failure message="Cannot PUT /kisenyi/table/mkdwchix" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /kisenyi/table/mkdwchix
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CreateTableTest$14$1.done(CreateTableTest.java:644)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CreateTableTest$14.done(CreateTableTest.java:638)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CreateTableTest.createTablestudent1(CreateTableTest.java:627)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CreateTableTest" name="createTableHostel" time="0.351">
    <failure message="Cannot PUT /kisenyi/table/shkybsut" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /kisenyi/table/shkybsut
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CreateTableTest$13.done(CreateTableTest.java:605)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CreateTableTest.createTableHostel(CreateTableTest.java:599)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CreateTableTest" name="createTableCustom7" time="0.369">
    <failure message="Cannot PUT /kisenyi/table/cazudhjx" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /kisenyi/table/cazudhjx
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CreateTableTest$18.done(CreateTableTest.java:770)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CreateTableTest.createTableCustom7(CreateTableTest.java:764)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CreateTableTest" name="deleteUnderscoreTable" time="0.001"/>
  <testcase classname="io.cloudboost.CreateTableTest" name="createTableCustom" time="0.388">
    <failure message="Cannot PUT /kisenyi/table/zpfbymbd" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /kisenyi/table/zpfbymbd
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CreateTableTest$9.done(CreateTableTest.java:454)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CreateTableTest.createTableCustom(CreateTableTest.java:448)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CreateTableTest" name="deleteCreatestudent_4" time="0.351"/>
  <testcase classname="io.cloudboost.CreateTableTest" name="createTableCustom3" time="0.369">
    <failure message="Cannot PUT /kisenyi/table/cewwpcxx" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /kisenyi/table/cewwpcxx
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CreateTableTest$17.done(CreateTableTest.java:740)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CreateTableTest.createTableCustom3(CreateTableTest.java:734)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CreateTableTest" name="sequentialTests_0" time="0.361">
    <failure message="Cannot PUT /kisenyi/table/cifrnjxq" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /kisenyi/table/cifrnjxq
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CreateTableTest$1.done(CreateTableTest.java:31)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CreateTableTest.sequentialTests_0(CreateTableTest.java:25)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CreateTableTest" name="deleteCreateUnderscore" time="0.341">
    <failure type="junit.framework.AssertionFailedError:">junit.framework.AssertionFailedError: null
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.Assert.assertTrue(Assert.java:20)
	at junit.framework.Assert.assertTrue(Assert.java:27)
	at io.cloudboost.CreateTableTest$5.done(CreateTableTest.java:249)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CreateTableTest.deleteCreateUnderscore(CreateTableTest.java:244)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
</testsuite>